extensions/jupyter-adapter/src/RuntimeClientAdapter.ts

```diff
@@ -12,6 +12,11 @@ import { JupyterCommMsg } from './JupyterCommMsg';
 import { JupyterCommClose } from './JupyterCommClose';
 import { PromiseHandles, delay, uuidv4 } from './utils';
 
+interface IRuntimeClientOutput {
+	data: { [key: string]: any };
```

Comment:
```suggestion
	data: Record<string, any>;
```
Maybe to match the style of using a helper generic for `Array<>` beneath?

Reply:
Oh I guess that was the pattern from before, same argument could be made to do `buffers?: Uint8Array[];` in that case.